src/interactive/modal.story.jsx

```diff
@@ -39,6 +49,73 @@ storiesOf('Modal', module)
 					>
 						{content}
 					</Modal>
+					<div style={iconSpriteStyle} dangerouslySetInnerHTML={{__html: iconSprite}} />
+				</InfoWrapper>
+			</div>
+		)
+	)
+	.addWithInfo(
+		'has hero - color',
+		'This is the component with an extended header.',
+		() => (
+			<div style={wrapperStyle}>
+				<InfoWrapper>
+					<Modal
+						onDismiss={onDismiss}
+						heroInverted
+						heroBgColor='rgb(55,30,172)'
```

Comment:
We have an open ticket for this. It's not an urgent need, but it would definitely be nice to have later.